CHANGELOG.md

```diff
@@ -18,6 +18,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ---
 
 ## [Unreleased] 
+### Added 
+- Device Authorization Grant flow for input constrained devices needing OAuth
+credentials from a non-Google OIDC provider. Users with Google providers can 
+force the Device Authorization Grant flow (rather than the default OOB flow)
+by setting the environment variable `DEVICE=1`.
```

Comment:
Why do we need to do this?

It would be easy to have three boolean flags `--console`, `--oob` and `--device`. With `--console` with what we're currently doing on this PR, with `--device` with force the device flow, and with `--oob` we force the OOB flow. You will be able to set variables like `STEP_DEVICE=true` or edit the ca.json